src/main/java/org/broadinstitute/consent/http/models/support/SupportRequest.java

```diff
@@ -0,0 +1,74 @@
+package org.broadinstitute.consent.http.models.support;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.broadinstitute.consent.http.enumeration.SupportRequestType;
+
+/**
+ * Represents Request object for the Zendesk API
+ * <a href="https://developer.zendesk.com/api-reference/ticketing/tickets/ticket-requests/">Requests API Reference</a>
+ */
+public class SupportRequest {
+
+  private SupportRequester requester;
+  private String subject;
+  private List<CustomRequestField> customFields;
+  private SupportRequestComment comment;
+  private final long ticketFormId = 360000669472L;
```

Comment:
It is odd having a field that's only here for serialization purposes, but I guess that's the nature of this class. It's good that there's a test for this to prevent someone converting it to a constant field.